lib/services/resource.go

```diff
@@ -181,6 +181,8 @@ func ParseShortcut(in string) (string, error) {
 		return types.KindDatabaseService, nil
 	case types.KindLoginRule, types.KindLoginRule + "s":
 		return types.KindLoginRule, nil
+	case types.KindSAMLIdPServiceProvider, types.KindSAMLIdPServiceProvider + "s", "saml_sp", "saml_sps":
```

Comment:
Are we expected to be able to create `KindSAMLIdPServiceProvider` from `tctl create` ? Or list them using `tctl get saml_sps` ?

Reply:
Yes, both of those are expected.